e2e-tests/tfm/cypress/support/trade-finance-manager-ui/submit-deal-cancellation.js

```diff
@@ -0,0 +1,18 @@
+import relative from '../../e2e/relativeURL';
+import * as caseDealPage from '../../e2e/pages/caseDealPage';
+import * as checkDetailsPage from '../../e2e/pages/deal-cancellation/check-details';
+
+export const submitDealCancellation = ({ dealId, effectiveDate }) => {
```

Comment:
Can we add some documentation please - I think this might be copied from another pending PR, in which case if the other PR is merged first, no need

Reply:
added